nservicebus/messaging/databus/index_obsolete_core_[9.1,).partial.md

```diff
@@ -0,0 +1,2 @@
+> [!WARNING]
+> The `DataBus` feature is available as a [dedicated NuGet package](/nservicebus/messaging/databus/). The API documented on this page will continue to work for NServiceBus Version 9 but it will hint about its upcoming obsoletion with the following warning: *The DataBus feature is released as a dedicated 'NServiceBus.ClaimCheck.DataBus' package.*.
```

Comment:
Todo:

make sure the link to the nuget package docs is correct
```suggestion
> The `DataBus` feature is available as a [dedicated NuGet package](/nservicebus/messaging/databus/). The API documented on this page will continue to work for NServiceBus Version 9 but it will hint about its upcoming obsoletion with the following warning: *The DataBus feature is released as a dedicated 'NServiceBus.ClaimCheck.DataBus' package.*.
```